sdk/avs/Azure.ResourceManager.Avs/tests/Scenario/WorkloadNetworkVmGroupCollectionTest.cs

```diff
@@ -47,34 +47,34 @@ public async Task GetCollectionOld()
             Assert.IsTrue(workloadNetworksList.Any());
         }
 
-        [TestCase, Order(2)]
+        [TestCase, Order(3)]
         [RecordedTest]
         public async Task GetResource()
         {
             WorkloadNetworkVmGroupCollection collection = getWorkloadNetworkResource().GetWorkloadNetworkVmGroups();
             WorkloadNetworkVmGroupResource result =  await collection.GetAsync(WORKLOAD_NETWORK_NAME);
-            Assert.AreEqual(result.Data.Name, WORKLOAD_NETWORK_NAME);
+            Assert.AreEqual(WORKLOAD_NETWORK_NAME, result.Data.Name);
         }
 
-        [TestCase, Order(3)]
+        [TestCase, Order(4)]
         [RecordedTest]
         public async Task resourceExisits()
```

Comment:
could we change this name to be capitalized?
```suggestion
        public async Task ResourceExisits()
```